docs/contrib/releases.md

```diff
@@ -6,6 +6,34 @@ We try to follow the [PEP 440](https://peps.python.org/pep-0440/) versioning sch
 
 ## Procedure
 
-1. Add an entry in [changelog.md](../changelog.md). Remember to give thanks to all the contributors!
-2. Use `tbump` to bump the version.
-3. Edit the release in GitHub. Copy the changelog entry into its description
+1. Commit following changes either as a PR to `main` branch or directly to `main`
+    * Add an entry in [changelog.md](../changelog.md). Remember to give thanks to all the contributors!
+    * Update `tbump` file to bump the version.
+
+        ```toml
+        [version]
+        # Set the new version as `current`
+        current = "1.2.3"
+        ```
+
+    * Update `version` file to bump the version.
+
+2. Create a tag for the new version, `v1.2.3`, from the latest of `main` that includes the above changes.
```

Comment:
You don't need to do that if you use `tbump`, it does that for you.

See https://github.com/your-tools/tbump#screenshot.

Reply:
Ahh... Learned something new today. I updated things manually in the last release. I'll make an update later appropriate steps.

Reply:
I think I know the basics of `tbump` tool now. Pushed an update with some details and hopefully clarifies the release process. Please review.